.travis.yml

```
@@ -11,14 +11,14 @@ jobs:
       script:
         - npm run test
         - ./cc-test-reporter after-build -t lcov --debug --exit-code $TRAVIS_TEST_RESULT
-        - xvfb-run npm run test:e2e:storybook:ci
+        - xvfb-run -a npm run test:e2e:storybook:ci
```

```suggestion
        - xvfb-run npm run test:e2e:storybook:ci
```

---

Having discussed this with @benjaminhobbs, I will resolve this comment since the `-a` is added to address the flakiness of Travis builds
```suggestion
        - xvfb-run -a npm run test:e2e:storybook:ci
```